src/Symfony/Component/Form/Extension/Validator/ValidatorTypeGuesser.php

```diff
@@ -168,7 +168,16 @@ public function guessRequiredForConstraint(Constraint $constraint)
             case 'Symfony\Component\Validator\Constraints\NotNull':
             case 'Symfony\Component\Validator\Constraints\NotBlank':
             case 'Symfony\Component\Validator\Constraints\IsTrue':
-                return new ValueGuess(true, Guess::HIGH_CONFIDENCE);
+                $groups = ['Default'];
+
+                if (\func_num_args() > 1 && $class = func_get_arg(1)) {
```

Comment:
We should now document this somehow. Do we want to add the argument to the signature on v6?